File: andming/src/main/java/com/andming/service/BrandService.java
package com.andming.service;

import com.andming.domain.TbBrand;
import com.andming.domain.entity.PageResult;

import java.util.List;

public interface BrandService {

    List<TbBrand> findAll();

    //添加
    void add(TbBrand brand);

    //删除
    void del(long[] ids);

    //根据主键查询
    TbBrand findOne(long id);

    //修改
    void updata(TbBrand brand);

    PageResult findPage(Integer pageNum, Integer pageSize);
}

File: andming/src/main/webapp/js/service/BrandService.js
app.service("brandService",function ($http) {
    // 查询所有
    this.findAll=function () {
        return $http.get("../brand/findAll.do")
    };
    // 分页查询
    this.findPage=function (pageNum,pageSize) {
        return $http.get("../brand/findPage.do?pageNum="+pageNum+"&pageSize="+pageSize);
    }
    // 条件查询
    this.findOne=function (id) {
        return $http.get("../brand/findOne.do?id="+id)
    }
    // 添加
    this.add=function (entity) {
        return  $http.post("../brand/add.do",entity)
    }
    // 修改
    this.update=function (entity) {
        return  $http.post("../brand/update.do",entity)
    }
    // 删除
    this.dele=function (ids) {
        return $http.get("../brand/delete.do?ids="+ids);
    }
});